index.html

```diff
@@ -71,7 +72,7 @@
     </div>
 
     <div class="grid-item2">
-      <p style="color: #67a435;margin-top: 15px;" class="griditemmobp1">
+      <p  class="gls griditemmobp1">
```

Comment:
have meaningful names

Reply:
updated the Class Name "gls" to "playbuttongamesection".

Reply:
updated the Class Name "gls" to "playbuttongamesection".